GooglePayment/src/test/java/com/braintreepayments/api/GooglePaymentUnitTest.java

```diff
@@ -180,19 +180,57 @@ public void requestPayment_whenPayPalDisabledInConfiguration_tokenizationPropert
         JSONArray allowedPaymentMethods = getPaymentDataRequestJsonSentToGooglePayment(fragment)
                 .getJSONArray("allowedPaymentMethods");
 
-        assertEquals(1, allowedPaymentMethods.length());
-        assertEquals("CARD", allowedPaymentMethods.getJSONObject(0)
+        assertEquals(2, allowedPaymentMethods.length());
+        assertEquals("PAYPAL", allowedPaymentMethods.getJSONObject(0)
+                .getString("type"));
+        assertEquals("CARD", allowedPaymentMethods.getJSONObject(1)
                 .getString("type"));
     }
 
     @Test
-    public void requestPayment_whenPayPalConfigurationLacksClientId_tokenizationPropertiesLackPayPal() throws JSONException {
+    public void requestPayment_usesGooglePaymentConfigurationClientId() throws JSONException {
+         TestConfigurationBuilder configuration = new TestConfigurationBuilder()
+                .googlePayment(new TestConfigurationBuilder.TestGooglePaymentConfigurationBuilder()
+                        .environment("sandbox")
+                        .googleAuthorizationFingerprint("google-auth-fingerprint")
+                        .paypalClientId("paypal-client-id-for-google-payment")
+                        .supportedNetworks(new String[]{"visa", "mastercard", "amex", "discover"}))
+                 .paypal(new TestConfigurationBuilder.TestPayPalConfigurationBuilder(true)
+                         .clientId("paypal-client-id-for-paypal"))
+                .withAnalytics();
+
+        BraintreeFragment fragment = getSetupFragment(configuration);
+        GooglePaymentRequest googlePaymentRequest = mBaseRequest
+                .googleMerchantName("google-merchant-name-override");
+
+        GooglePayment.requestPayment(fragment, googlePaymentRequest);
+
+        JSONArray allowedPaymentMethods = getPaymentDataRequestJsonSentToGooglePayment(fragment)
+                .getJSONArray("allowedPaymentMethods");
+
+        JSONObject paypal = allowedPaymentMethods.getJSONObject(0);
+
+        assertEquals("paypal-client-id-for-google-payment",
+                paypal.getJSONObject("parameters")
+                        .getJSONObject("purchase_context")
+                        .getJSONArray("purchase_units")
+                        .getJSONObject(0)
+                        .getJSONObject("payee")
+                        .getString("client_id"));
+
+        assertEquals("paypal-client-id-for-google-payment",
+                paypal.getJSONObject("tokenizationSpecification")
+                        .getJSONObject("parameters")
+                        .getString("braintree:paypalClientId"));
+    }
+
+    @Test
+    public void requestPayment_whenGooglePaymentConfigurationLacksClientId_tokenizationPropertiesLackPayPal() throws JSONException {
```

Comment:
Should this test include an `assertEmpty` type test on the `paypal-client-id-for-google-payment` key?

Reply:
Added a check, thanks!